### DISCUSSION TITLE
Add Game Request

### DISCUSSION THREAD

**Discussion by MrFunny8** [GoldHEN/GoldHEN_Cheat_Repository#368] [Cheat requests]
Tony Hawk Pro Skater 1+2 (CUSA17922 or CUSA20464)
Crash Team Racing Nitro Fueled (CUSA14876 or CUSA13795)
Destroy All Humans! 2 Reprobed (CUSA36164 or CUSA36167)
Destroy All Humans Clone Carnage CUSA32947 v1.03
LA Noire (CUSA09084 or CUSA09172)
LA Noire The VR Case Files CUSA15490 v1.02

**Comment by 01sali** [comment]
Returnal_PS5-PPSA01285_02.000.003

**Comment by k0rk1969** [comment]
Sniper Elite Resistance Deluxe (CUSA44067)

**Comment by Don-Keballs** [comment]
Batman Arkham Knight (CUSA00133) For update v01.18

**Comment by adddaar** [comment]
CUSA32137 Like A Dragon Infinite Wealth Ultimate Edition v1.16

**Comment by Shadotterdan** [comment]
CUSA-25458 r-type final 2 v1.18

**Comment by Brulux98** [comment]
Cod AW all versions
Cod Ghosts all versions
